### PR TITLE
resizable debugger window

### LINKED ISSUE #5222: What if there was a Re sizable debug window?
### Why this would be helpful/solve a problem

When I want to code something and check on the fps at the same time, I can't because the huge screen blocks the code making it difficult to work on my project
Untitled

### What this change does

It would allow the debug window to resize

### Possible alternatives

It could also be a bar at the bottom that you can click on and it will appear on the code

### Demonstration(s) and additional context

Untitled
Or
Untitled

### PR DESCRIPTION
**Pull request by Platformer444**
Resolves #5222 

### Changes
This makes the debugger window resizable

### Tests
Works fine

### CODE REVIEW

File: addons/debugger/style.css [hunk 1]
```diff
@@ -35,6 +35,9 @@
   background-color: white;
   width: 565px;
   height: 25rem;
+  resize: both;
+  min-width: 565px;
+  min-height: 25rem;
```
Comment (RedGuy12) [inline addons/debugger/style.css:40]: What if you want to make it smaller? I think that's the main want currently, it's quite big

Comment (Platformer444) [reply addons/debugger/style.css:40]: I think I should make this thing a setting and those values customizable

Comment (WorldLanguages) [reply addons/debugger/style.css:40]: It doesn't make sense to customize the minimum dimensions.
The minimum dimensions should be the lowest ones where the buttons still fit and our CSS doesn't break.

Comment (Platformer444) [reply addons/debugger/style.css:40]: So **I** have to manually put that values

Comment (BroJac5246) [reply addons/debugger/style.css:40]: > So **I** have to manually put that values

Correct.

Comment (Platformer444) [reply addons/debugger/style.css:40]: @WorldLanguages Is it a good size (height: 335px and width: 483px)?

image

Comment (WorldLanguages) [reply addons/debugger/style.css:40]: @Platformer444 Personally I'd round the width to 450px and height to 300px

Comment (BroJac5246) [reply addons/debugger/style.css:40]: > @Platformer444 Personally I'd round the width to 450px and height to 300px

Does that work? You're rounding down. (Just making sure before something theoretically breaks.)

Also, why does rounding matter? The minimum dimensions should be the minimum dimensions possible, not some number evenly divisible by 50 just for the sake of being evenly divisible by 50. The difference between what's allowed and what's possible might matter to some people.

Comment (WorldLanguages) [reply addons/debugger/style.css:40]: > You're rounding down.

Sorry, you're right. The min width can't be 450px

Comment (Platformer444) [reply addons/debugger/style.css:40]: @WorldLanguages Is this good now(height: 300px and width: 483px)?
image

Comment (mxmou) [reply addons/debugger/style.css:40]: Does the minimum width work in languages with longer button labels?

Comment (Platformer444) [reply addons/debugger/style.css:40]: > Does the minimum width work in languages with longer button labels?

Oh didn't see that

Comment (Samq64) [reply addons/debugger/style.css:40]: The debugger window doesn't seem to be translated.

Comment (mxmou) [reply addons/debugger/style.css:40]: @Samq64 It is - you need to reload the page after changing the language.
image

**Comment by WorldLanguages** [comment]
Related: I believe the **current** (store release) version of debugger fails to account for longer strings in other languages.

**Comment by mxmou** [comment]
> Related: I believe the **current** (store release) version of debugger fails to account for longer strings in other languages.

The current width (which is also the minimum width in this PR) is wide enough in all languages except Portuguese.
image

The minimum width proposed in the discussion above is not - it overflows in German, for example:
image

A possible solution is to calculate the minimum width in the userscript based on the width of the labels. Alternatively, the tabs could become scrollable if the window is too narrow.

**Comment by WorldLanguages** [comment]
@mxmou I think we should make it scrollable for now

**Comment by TheColaber** [comment]
closing due to inactive author. if anyone wants to implement wl's above comment, feel free to pr.

**Comment by WorldLanguages** [comment]
+1 from feedback, see: https://github.com/ScratchAddons/ScratchAddons/issues/5222#issuecomment-2303266162

**Pull request closed without merging**